include/psp2/kernel/modulemgr.h

```diff
@@ -37,23 +37,34 @@ typedef struct SceKernelSegmentInfo
 	SceUInt res;    //!< unused?
 } SceKernelSegmentInfo;
 
+typedef struct {
```

Comment:
I think the convention is to put the struct name after "typedef struct" too.

Reply:
`SceKernelLMOption`, `SceKernelULMOption` and `SceKernelModuleInfo` does not follow theses rules neither.
Should I also update them ?